.github/workflows/test.yml

```diff
@@ -5,11 +5,13 @@ on: [push, pull_request]
 
 jobs:
   lint:
```

Comment:
```suggestion
  test:
```

Reply:
It will run the linter tests too:
![image](https://user-images.githubusercontent.com/6695818/125643135-b79a55ac-2bf3-4f47-a086-e8962583f34c.png)

I guess it runs it because of the pretest script:
https://github.com/Ericsson/CodecheckerVSCodePlugin/blob/5988cde94bb5974003abb7f114e1b6879100d5cf/package.json#L71

Reply:
Indeed it does, in this case that's fine.